security-command-center/snippets/pom.xml

```diff
@@ -69,8 +70,8 @@
     <dependency>
       <groupId>com.google.truth</groupId>
       <artifactId>truth</artifactId>
-      <version>1.4.0</version>
+      <version>1.2.0</version>
```

Comment:
what is a reason for downgrading this package?

Reply:
It wasn't intentional. Thanks for the catch.